### DISCUSSION TITLE
Documentation on guest_token API needs more details

### DISCUSSION THREAD

**Discussion by sanjayssk** [apache/superset#22734] [Q&A / Help]
There is no issue type on improving the documentation. So adding it under this category.

## Screenshot

[![guest_tokenAPI](https://user-images.githubusercontent.com/20315846/181871039-70417081-de42-4d75-a5bf-6c4fb974a6e3.PNG)]

## Description

Please see the screenshot of input payload. There is no information on what types are available to use in the resources object. If I go by just the single example included in the sources, the only type it uses is "dashboard". What other resource types are available, for example, "chart"? 

Further, it is not clear if the id is the id within a dashboard URL or the embedded id obtained from "Embed Dashboard" feature? The example seems to use an id from the URL which is just 1, 2, 3 etc.

**Comment by tys1128** [comment]
I have the same confused before, but then I found the document in Preset is more detailed. https://docs.preset.io/docs/step-2-deployment

**Comment by rusackas** [comment]
Pinging @tugceoz and @lilykuang in case they can advise on additional details to flesh out the documentation here. Meanwhile I'll close this out as an issue and move it to a Discussion since it's not a bug, per se, but more of a (perfectly reasonable) documentation request. We'll be more than happy to review PRs if anyone reading this is willing and able to open a documentation PR.
